airbyte-integrations/connectors/source-mssql/src/test-integration/java/io/airbyte/integrations/source/mssql/AbstractMssqlSourceDatatypeTest.java

```diff
@@ -344,6 +344,14 @@ protected void initTests() {
 
     }
 
+    addDataTypeTestData(
+        TestDataHolder.builder()
+            .sourceType("int")
+            .airbyteType(JsonSchemaType.INTEGER)
+            .addInsertValues("null", "1234", "7878")
+            .addExpectedValues(null, "1234", "7878")
+            .createTablePatternSql("CREATE TABLE %1$s(%2$s INTEGER DEFAULT ((7878)) NULL, %3$s %4$s)")
```

Comment:
??? What's this `((7878)) NULL` ?
I think we need more than one test:
1) column DEFAULT NULL, set to NULL, set to XXX(not null), and set to nothing.
2) column DEFAULT XXX, set to NULL, set to XXX, set to YYY and set to nothing.
so a total of at least 7 rows.
The resulting data should be NULL, XXX, NULL, NULL, XXX, YYY, XXX

Reply:
I'm very concerned that test case4 is going to be wron.

Reply:
`INTEGER NULL DEFAULT xxx` feels easier to read and understand